core/templates/pages/exploration-editor-page/editor-tab/services/responses.service.ts

```diff
@@ -334,7 +334,7 @@ export class ResponsesService {
   }
 
   updateActiveAnswerGroup(
-      updates: AnswerGroup,
+      updates: AnswerGroup | object,
```

Comment:
Why `object`?

Reply:
I will handle this while strict typing this file. This file is not yet fully strict typed.

Reply:
done